modules/appeals_api/config/schemas/v2/contestable_issues_headers.json

```diff
@@ -28,6 +28,13 @@
         "X-VA-Receipt-Date": {
           "description": "(yyyy-mm-dd) Date to limit the contestable issues",
           "type": "string", "format": "date"
+        },
+        "X-VA-ICN": {
+          "type": "string",
+          "description": "Veteran's ICN",
+          "pattern": "^[0-9]{10}V[0-9]{6}$",
+          "minLength": 17,
+          "maxLength": 17
```

Comment:
Since the `X-VA-ICN` header is excluded from the headers to be validated when the feature flag is turned off, these header requirements can stay here.